src/js/player.js

```diff
@@ -453,17 +446,8 @@ class Player extends Component {
     const playerOptionsCopy = mergeOptions(this.options_);
 
     // Load plugins
-    if (options.plugins) {
-      const plugins = options.plugins;
 
-      Object.keys(plugins).forEach(function(name) {
-        if (typeof this[name] === 'function') {
-          this[name](plugins[name]);
-        } else {
-          throw new Error(`plugin "${name}" does not exist`);
-        }
-      }, this);
-    }
+    this.pluginif(options);
```

Comment:
I think it would be ideal if this function had a name that implied more about what it is doing instead of saying that it involves if statements. This could be as simple as renaming it to "loadPlugins" which is what the developers wrote in a comment above the original code.